frontend/public/components/modals/_modals.scss

```diff
@@ -13,3 +13,44 @@
     width: 100%;
   }
 }
+
+.modal-body {
+  display: flex;
+  flex: 1 1 auto;
+  flex-direction: column;
+  overflow-y: auto;
+  -webkit-overflow-scrolling: touch; // enable momentum scrolling in mobile Safari
+
+  @media(min-width: $screen-sm-min) {
+    max-height: calc(100vh - 162px); // 177px = 30 (margin-top) + 43 (modal-header) + 15 (modal-footer margin-top) + 44 (modal-footer) + 30 (margin-bottom)
```

Comment:
`vh` units are problematic because of mobile.  Did this test okay? 

`162px` doesn't match the `177px` in the comment.  I assume they should?